src/mono/wasm/README.md

```diff
@@ -50,25 +52,56 @@ The latest engines can be installed with jsvu (JavaScript engine Version Updater
 
 `npm install jsvu -g`
 
-* Run jsvu:
+* Run jsvu and install `v8`, `SpiderMonkey`,or `JavaScriptCore` engines:
 
 `jsvu`
 
 Add `~/.jsvu` to your `PATH`:
 
 `export PATH="${HOME}/.jsvu:${PATH}"`
 
+### Windows
+
+* Install node/npm from https://nodejs.org/en/ and add its npm and nodejs directories to the `PATH` environment variable
+
+* * Install jsvu with npm:
+
+`npm install jsvu -g`
+
+* Run jsvu and install `v8`, `SpiderMonkey`,or `JavaScriptCore` engines:
+
+`jsvu`
+
+* Add `~/.jsvu` to the `PATH` environment variable
+
 ## Libraries tests
 
 Library tests can be run with js engines: `v8`, `SpiderMonkey`,or `JavaScriptCore`:
 
+### Mac
+
 * `v8`: `make run-tests-v8-$(lib_name)`
 * SpiderMonkey: `make run-tests-sm-$(lib_name)`
 * JavaScriptCore: `make run-tests-jsc-$(lib_name)`
 * Or default: `make run-tests-$(lib_name)`. This runs the tests with `v8`.
 
 For example, for `System.Collections.Concurrent`: `make run-tests-v8-System.Collections.Concurrent`
 
+### Windows
+
+To run the libraries tests on windows, we have `RunTests` target. Without setting additional properties, it will run tests for all libraries on `V8` and `SpiderMonkey` engines:
+
+`dotnet build /t:RunTests wasm.proj`
+
+* `Library` property can be used to specify which library tests to run
+* `VESngine` property can be used to specify which engine to use
+
+For example to run tests for `System.Collections.Concurrent` on `V8` engine, use:
+
+`dotnet build /t:RunTests /p:Library="System.Collections.Concurrent" /p:VSEngine="v8" wasm.proj`
```

Comment:
```suggestion
`dotnet build /t:RunTests /p:Library="System.Collections.Concurrent" /p:JSEngine="v8" wasm.proj`
```

Reply:
not sure we need this special target, the documented way is using `./build.sh libs.tests -test -os browser` to build and run all library tests: https://github.com/dotnet/runtime/blob/main/docs/workflow/testing/libraries/testing.md#testing-libraries

You could pass down the property to specify the JS engine via `/p:JSEngine=v8`

Reply:
I added the target to have something similar to https://github.com/dotnet/runtime/blob/main/src/mono/wasm/Makefile#L179-L187

Reply:
yeah like I said that can be achieved by `build.cmd libs.tests -test -os browser /p:JSEngine=v8`, which is consistent with how other parts of the repo do it. I'd rather not add another custom target.

Reply:
I think you are right. The individual library run has similar verbosity to running it directly too. I have updated the README and removed the target.